src/test/java/com/facebook/openwifirrm/optimizers/tpc/RandomTxPowerInitializerTest.java

```diff
@@ -63,16 +66,32 @@ private static DataModel createModel() {
 		return dataModel;
 	}
 
+	/**
+	 * Return the first value provided by Random, for each key seed
+	 */
+	private static Map<Integer, Integer> mapRngSeedToFirstValue() {
+		Map<Integer, Integer> map = new HashMap<>();
+		map.put(0, 2);
```

Comment:
I would prefer TreeMap in tests if you intend to keep this map, so that the orders are reserved.

Reply:
good to know! first time writing java :p